src/main/java/com/flippingutilities/model/HistoryManager.java

```diff
@@ -69,11 +65,35 @@
 	@SerializedName("pIB")
 	private int itemsBoughtThroughCompleteOffers;
 
+	//If this HistoryManager belonged to the FlippingItem representing a guthan set and the user had created a
+	//combination flip out of a guthan set and its pieces, the combination flip would show up here.
+	@Getter
+	private List<CombinationFlip> personalCombinationFlips = new ArrayList<>();
```

Comment:
Yea the naming is kind of confusing. Need a bit of help on that. Just to clarify with an example:

You have a FlippingItem that represents a Blowpipe (contains all your trades for blowpipes) and another for Tanzanite Fang (contains all your trades for TFs).

You create a combination flip using an offer event for the Blowpipe and an offer event for the TF. This combination flip now exists in the HistoryManager of the **Blowpipe** FlippingItem in the `personalCombinationFlips` field. This same combination flip ALSO exists in the `parentCombinationFlips` field in the **TF** HistoryManager. It is called `parentCombinationFlips` because it stores combination flips that other items are the "parents" of and not this item.